Ginger/GingerCoreNET/Run/Context.cs

```diff
@@ -90,11 +95,25 @@ public Activity Activity
                 if (mActivity != value)
                 {
                     mActivity = value;
+                    if (mActivity != null)
```

Comment:
Context should not have any properties calculations or dependency between the properties
the only things to add to Context is "Agent"  and "Platform" (ePlatformType).

all actual values calculations should be done in NewAutomatePage to update Context

Reply:
Done the requested changes